tensorflow/compiler/xla/service/pattern_matcher.h

```diff
@@ -1454,6 +1480,13 @@ class HloInstructionPatternOperandImpl {
       EXPLAIN << "\nin operand " << operand_index_;
       return false;
     }
+    if (option.single_user_operands &&
+        inst->operand(operand_index_)->user_count() != 1) {
```

Comment:
What about use count being less than one, do we want to match those as well? I'd imagine we do (e.g. computation root?)

Reply:
That's a good point. Currently, the top-level operation in the pattern can have any number of users (see the comment in pattern_matcher.h), and it seems that intermediates/operands by definition have at least one user.